commons/src/main/java/io/hyscale/commons/logger/TableFields.java

```diff
@@ -18,7 +18,8 @@
 public enum TableFields {
 
 	SERVICE("Service Name", 30), STATUS("Status", 15), AGE("Age"), REASON("Reason"), MESSAGE("Message"),
-	SERVICE_ADDRESS("Service Address", 40), SERVICE_ADDRESS_LARGE("Service Address", 75), REPLICA_NAME("Replica name", 50), INDEX("Index", 7);
+	SERVICE_ADDRESS("Service Address", 40), SERVICE_ADDRESS_LARGE("Service Address", 75), REPLICA_NAME("Replica name", 50), 
+	INDEX("Index", 7), NAMESPACE("Namespace", 30), APPS("Apps", 30), SERVICES("Services", 40);
```

Comment:
Make it as a constant, so that we just refer the constant everywhere. Difficult to change in regex and here always